package.json

```diff
@@ -19,11 +19,6 @@
   },
   "homepage": "https://github.com/SAP/styleguides#readme",
   "dependencies": {
-    "@asciidoctor/cli": "^3.5.0",
-    "@asciidoctor/core": "^2.2.6",
-    "asciidoc-link-check": "^1.0.15",
-    "asciidoctor-kroki": "^0.15.4",
-    "asciidoctor-pdf": "1.0.0-alpha.16",
-    "asciidoctor-plantuml": "^1.5.0"
+    "mermaid":"^8.11.0",
```

Comment:
Why do we need a dependency on mermaid? We do not want to process mermaid content during build.
We only need a markdown-linkcheck here.

Reply:
Yes, mermaid is removed. Do we also delete the [asciidoc-link-check-config.json](https://github.com/priyasingh199885/styleguides/blob/main/asciidoc-link-check-config.json) file?